### DISCUSSION TITLE
Python bindings to choco-solver

### DISCUSSION THREAD

**Discussion by cprudhom** [chocoteam/choco-solver#907] [Polls]
Since the Choco-solver library is written in Java, we use [GraalVM](https://www.graalvm.org/) in order to provide a native C api to it. Then it is possible to provide a Python bindings for the library.

**Comment by dimitri-justeau** [comment]
This is an excellent perspective to broaden the potential range of users (e.g. from biology, data science, etc) of Choco. GraalVM also provides better performances, handles longs as fast as ints, and offers an extensive and innovative ecosystem that might be very interesting in the future.

**Comment by cprudhom** [comment]
I will try to provide a POC in the next weeks, just to check how convenient it is and how easy it can be done automatically.
